backend.py

```diff
@@ -1180,10 +1202,11 @@ def check_efibootmgr_err(rc, err, install_type, err_type):
         efi = "EFI/xenserver/grubx64.efi"
     else:
         raise RuntimeError("Failed to find EFI loader")
-    rc, err = util.runCmd2(["chroot", mounts['root'], "/usr/sbin/efibootmgr", "-c",
-                            "-L", branding['product-brand'], "-l", '\\' + efi.replace('/', '\\'),
-                            "-d", disk, "-p", str(boot_partnum)], with_stderr=True)
-    check_efibootmgr_err(rc, err, install_type, "Failed to add new efi boot entry")
+    for disk in physical_disks:
```

Comment:
I would expect that before the end of installation, the installer should check and wait until the RAID is consistent and fully synchronized. Maybe that would fix the issue writing boot entries?